types/types.go

```diff
@@ -3,3 +3,8 @@ package types
 type DataStore struct {
 	Datadir string
 }
+
+type Key struct {
```

Comment:
What's this type for (add a description please). Shouldn't this be part of scrutinizer types?

Reply:
I'll add a description. I think it should be moved to api, not scrutinizer, because it's used just for sending keys over json and including an index with each key.